tests/test_server.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import unittest
+import json
+
+from unittest import mock
+
+
+class TestServer(unittest.TestCase):
+    MockedRequestHandler = None
+    srv = None
+    m = None
+    m_eof = None
+
+    def setUp(self):
+        with mock.patch('builtins.open', mock.mock_open(), create=True):
+            with mock.patch('pickle.load', mock.Mock(), create=True):
+                import server
```

Comment:
Hm, it shouldn't trigger https://github.com/mushorg/tanner/blob/master/server.py#L26 as it is in the scope of the class and the import should only run everything in the module scope. But I might be wrong.